### ISSUE TITLE
java.lang.IllegalStateException: Duplicate RPC method textDocument/semanticHighlighting

### ISSUE THREAD

**Issue by andre2007** [gtache/intellij-lsp#61]
With recent update 1.5.1 this plugin stopped to work:

```
java.lang.IllegalStateException: Duplicate RPC method textDocument/semanticHighlighting.
	at org.eclipse.lsp4j.jsonrpc.services.ServiceEndpoints.lambda$getSupportedMethods$0(ServiceEndpoints.java:113)
	at org.eclipse.lsp4j.jsonrpc.services.AnnotationUtil.findRpcMethods(AnnotationUtil.java:66)
	at org.eclipse.lsp4j.jsonrpc.services.ServiceEndpoints.getSupportedMethods(ServiceEndpoints.java:90)
	at org.eclipse.lsp4j.jsonrpc.services.ServiceEndpoints.getSupportedMethods(ServiceEndpoints.java:82)
	at org.eclipse.lsp4j.jsonrpc.Launcher$Builder.getSupportedMethods(Launcher.java:370)
	at org.eclipse.lsp4j.jsonrpc.Launcher$Builder.createJsonHandler(Launcher.java:381)
	at org.eclipse.lsp4j.jsonrpc.Launcher$Builder.create(Launcher.java:305)
	at org.eclipse.lsp4j.launch.LSPLauncher.createClientLauncher(LSPLauncher.java:106)
	at com.github.gtache.lsp.client.languageserver.wrapper.LanguageServerWrapperImpl.start(LanguageServerWrapperImpl.scala:291)
	at com.github.gtache.lsp.client.languageserver.wrapper.LanguageServerWrapperImpl.connect(LanguageServerWrapperImpl.scala:169)
	at com.github.gtache.lsp.PluginMain$.$anonfun$editorOpened$2(PluginMain.scala:135)
	at com.github.gtache.lsp.PluginMain$.$anonfun$editorOpened$2$adapted(PluginMain.scala:132)
	at scala.Option.foreach(Option.scala:257)
	at com.github.gtache.lsp.PluginMain$.$anonfun$editorOpened$1(PluginMain.scala:132)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:314)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

I use this language protocol: https://github.com/d-language-server/dls
and have also plugin Intellij D Language installed.
This combination worked with previous version of intellij-lsp

**Comment by gtache** [comment]
Duplicate of #60, it's fixed in 1.5.2

**Comment by andre2007** [comment]
Thanks a lot. I can confirm, 1.5.2 fixes the issue.
